cli/api/daemon.go

```diff
@@ -111,6 +112,7 @@ type daemon struct {
 	tokenExpiration  time.Duration
 
 	facade *facade.Facade
+	ssm    *servicestatemanager.BatchServiceStateManager
```

Comment:
Oops, Argument 2 isn't really valid here.  We could still have the interface here as long as NewBatchServiceStateManager returns the impl and not the interface.